cactusbot/commands/magic/social.py

```diff
@@ -56,6 +56,12 @@ class Social(Command):
             return "Added social service {}.".format(service)
         elif response.status == 200:
             return "Updated social service {}".format(service)
+        elif response.status == 400:
+            json = await response.json()
+            if len(json["errors"].get("quote", {}).get("url", [])) > 0:
```

Comment:
A bit odd, yeah, but it's how the errors come in. Don't want the error handling exploding.